Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8.9.0
```

Comment:
If you are not using `make`, better use the Alpine one

e.g. 

```
FROM node:8-alpine
```


But you are using `make`, : (

Actually I'm still not get the idea why we use makefile in a node project, we are not using gcc/g++ etc.

Reply:
Thanks.
The make command is only used for building frontend files, not necessary to use make command, so I replace it with `npm run build` and switched to `node:8-alpine`